README.md

```diff
@@ -38,43 +36,65 @@ to the muncher container. If you do not want that, you can point muncher to a
 different Docker host via the `MUNCHER_DOCKER_HOST` and `MUNCHER_DOCKER_PORT`
 environment variables.
 
-###Available environment variables
+### Available environment variables
 
-* `MUNCHER_DOCKER_HOST`
+You can override these environment variables (configured in `config/config.js`) when starting the service.
+
+- `MUNCHER_DOCKER_HOST`
   Define a different Docker Remote API location to connect to. If omitted, muncher will try to connect to the local unix socket.
-* `MUNCHER_DOCKER_PORT`
+- `MUNCHER_DOCKER_PORT`
   Port for Docker Remote API
-* `MUNCHER_PORT`
+- `MUNCHER_PORT`
   Define on which Port muncher should listen. Defaults to `8080`.
-* `MUNCHER_MONGODB` __Required__
+- `MUNCHER_MONGODB` __Required__
   Location for the mongo db. Defaults to `mongodb://localhost/`. You will very likely need to change this.
-* `MUNCHER_MONGODB_COLLECTION`
+- `MUNCHER_MONGODB_COLLECTION`
   Which collection inside the mongo db should be used. Defaults to `muncher`.
-* `MUNCHER_BASEPATH`
+- `MUNCHER_BASEPATH`
   Base path for the compendia storage. Defaults to `/tmp/muncher`. If you want persistent compendia storage, you should point this to a separate volume.
-* `MUNCHER_APIKEY` __Recomended__
+- `MUNCHER_APIKEY` __Recomended__
   The API key that is required for posting a new compendium. It is highly recommended that you change this to a secure key. Defaults to `CHANGE_ME`.
 
 ### Full API service with docker-compose
 
-The o2r muncher only provides the main parts of the o2r web API. For example, serving the data (files) from the compendia is handled by o2r-contentbutler. To show a simple example implementation integrating both services, there is a additional compose file.
+The o2r muncher only provides the main parts of the o2r web API. For example, serving the data (files) from the compendia is handled by o2r-contentbutler. To show a simple example implementation integrating both services, there is a Docker compose configuration in the file `docker-compose/docker-compose.full.yml`.
 
-```
-cd docker-compose && docker-compose -f docker-compose.full.yml up
+```bash
+docker-compose -f docker-compose/docker-compose.full.yml up
 # after you're done, shutdown and delete all volumes (data):
 docker-compose -f docker-compose.full.yml down -v
```

Comment:
If you don't `cd` into the `docker-compose` folder, you wil also need to use `docker-compose -f docker-compose/docker-compose.full.yml up`.